lib/msal-core/src/ServerRequestParameters.ts

```diff
@@ -58,7 +59,8 @@ export class ServerRequestParameters {
         this.nonce = CryptoUtils.createNewGuid();
 
         // set scope to clientId if null
-        this.scopes = scopes? [ ...scopes] : [clientId];
+        this.scopes = scopes ? [ ...scopes] : [clientId];
```

Comment:
if scopes is already an array why spread it into a new one?

Reply:
To prevent mutation of the original object.

Reply:
Before if we directly assigned the value it would end up mutating the array passed in by the user.

Reply:
okay awesome!